### PR TITLE
About us

### PR DESCRIPTION
**Pull request by sok-leaphea**
add about us screen and design ui with team images
#record

https://github.com/user-attachments/assets/65510913-e3dd-4c93-8d61-dd4a7970df67

### CODE REVIEW

File: screens/AbooutUs.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+]; 
+
+const Item = ({ title, image }) => { 
+  return ( 
+    <View style={styles.item}> 
+      {image && <Image source={image} style={styles.image} />} 
+      <Text style={styles.title}>{title}</Text> 
+    </View> 
+  ); 
+};
```
Comment (heayem) [inline screens/AbooutUs.js]: Should be a profile component

**Review by heayem — CHANGES_REQUESTED** [review]

**Comment by heayem** [comment]
@sok-leaphea you try upload again
image

File: screens/AbooutUs.js (hunk 1 above)
Comment (sok-leaphea) [reply screens/AbooutUs.js]: I will make it to components later because I try to understand the structure first

**Review by heayem — APPROVED** [review]

File: components/ProfileList.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
+// components/TeamMemberList.js
```
Comment (heayem) [inline components/ProfileList.js:1]: should be no comment

**Review by heayem — CHANGES_REQUESTED** [review]

**Pull request merged**